tests/conftest.py

```diff
@@ -74,6 +74,18 @@ def backend(backend_name):
     yield get_backend(backend_name)
 
 
+@pytest.fixture
+def clear():
```

Comment:
https://docs.pytest.org/en/stable/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request